### ISSUE TITLE
Performance: Replicate the Node process

### ISSUE THREAD

**Issue by migramcastillo** [goldbergyoni/nodebestpractices#406]
TLDR: If you really need to use some blocking/heavy compute code in your application, you can’t use any of the recommendation about [non-blocking the event loop](https://nodejs.org/en/docs/guides/dont-block-the-event-loop/) and you have a managed server with more than a single CPU, you should consider using the Cluster Module native from NodeJS or a library like [PM2](http://pm2.keymetrics.io/). Cluster mode will allow you to take advantage of your CPU resources to replicate the Single Thread having availability when the Event Loop is blocked.

Otherwise: In high peak traffic your users could suffer some latency when the Event Loop is blocked and if you have a multi CPU managed server you will be wasting valuable resources.

I’m not including K8s because is an infrastructure matter.

PS: I'm not an English native speaker so any grammar mistake is welcome too!

@i0natan @js-kyle
